octree/include/pcl/octree/octree_iterator.h

```diff
@@ -524,6 +523,77 @@ namespace pcl
       };
 
     //////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+    /** \brief @b Octree iterator class
+     * \note This class implements a forward iterator for traversing octrees in a breadth-first manner.
+     *       It only returns the node at a given depth in the octree, if any.
+     * \ingroup octree
+     * \author Fabien Rozar (fabien.rozar@gmail.com)
+     */
+    template<typename OctreeT>
+      class OctreeBreadthFirstAtDepthIterator : public OctreeBreadthFirstIterator<OctreeT>
```

Comment:
Please shift this whole class definition left by two spaces to fix the alignment.
(I know the other classes in this file are misaligned. But we have a policy that the old code stays as it is, and the new code should follow the style guide.)